src/views/OtherTools.md

```diff
@@ -6,21 +6,36 @@
         <p>
                 <a href="http://monarch-exomiser-web-dev.monarchinitiative.org/exomiser">Exomiser</a>
                 is a Java program that functionally annotates variants
-                from whole-exome sequencing data starting from a VCF file (version 4).
-                The functional annotation code is based on Jannovar and uses UCSC
+                from whole-exome sequencing data starting from a VCF file (version 4) and a set 
+                of phenotypes encoded using the <a href="https://hpo.jax.org/app/">Human Phenotype 
+                Ontology</a>. The functional annotation of variants is based on Jannovar and uses UCSC
```

Comment:
we can do Ensembl UCSC or RefSeq on either hg19 or hg38

Reply:
Thanks, updated